crates/ruff/src/rules/wemake_python_styleguide/helpers/naming.rs

```diff
@@ -0,0 +1,78 @@
+use std::path::Path;
+
+use once_cell::sync::Lazy;
+use regex::Regex;
+
+const ALIAS_NAMES_WHITELIST: [&str; 7] = ["np", "pd", "df", "plt", "sns", "tf", "cv"];
+
+const UNUSED_PLACEHOLDER: char = '_';
+
+static UNUSED_VARIABLE_REGEX: Lazy<Regex> = Lazy::new(|| Regex::new(r"^_+$").unwrap());
+
+/// Checks whether the given ``name`` is unused.
+fn is_unused(name: &str) -> bool {
+    UNUSED_VARIABLE_REGEX.is_match(name)
+}
+
+/// Checks for too short names.
+pub(in crate::rules::wemake_python_styleguide) fn is_too_short_name(
+    name: &str,
+    min_length: usize,
+    trim: bool,
+) -> bool {
+    if ALIAS_NAMES_WHITELIST.contains(&name) {
+        return false;
+    }
+
+    if is_unused(name) {
+        return false;
+    }
+
+    let length = if trim {
+        name.trim_matches(UNUSED_PLACEHOLDER).len()
```

Comment:
We can use `trim_start_matches` here to avoid iterating over the whole name since we know that the `_` can only appear at the beginning of a name.

Reply:
I think in the original implementation the `trim=True` flag is meant to trim underscores from start and end (see [wemake_python_styleguide/logic/naming/logical.py#L103](https://github.com/wemake-services/wemake-python-styleguide/blob/a710b5f18ea35a7a09b713cc86cabc5a4fc6b25d/wemake_python_styleguide/logic/naming/logical.py#L103)). For example `z_` and `__z` should both be detected as too short variables names.